examples/user-management/sveltekit-user-management/src/routes/auth/confirm/+server.ts

```diff
@@ -0,0 +1,32 @@
+// src/routes/api/auth/confirm/+server.js
```

Comment:
I think `return`ing the `redirects` on lines 26 and 31 are incorrect?

Small thing, but the legacy path is still in the line 1 comment (`api`)